src/main/java/org/openrewrite/jenkins/github/InMemoryTeamNameValidator.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.jenkins.github;
+
+import org.openrewrite.internal.lang.Nullable;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * This is a simple stopgap to prevent the same issues from recurring.
+ * Ideally we'd have a near real-time view of the actual teams in GitHub.
+ * At the moment there are over 2500, so making an API call for each
+ * recipe run is likely to run into trouble.
```

Comment:
Not sure it this would be an option for you, but for the GitHub Advisories we have a [weekly scheduled job](https://github.com/openrewrite/rewrite-java-dependencies/blob/main/.github/workflows/advisories.yml) that downloads the latest and saves those as a resource, and creates a pull request if there are any changes. Could be a lightweight way of maintaining those as a not-so-static-list.

Reply:
I missed this comment earlier. That would be a very nice way to build on something that already exists. Thanks Tim!